docs/_guide/08-tools.md

```diff
@@ -72,31 +72,37 @@ ESLint is recommended for linting lit-html code. The following ESLint plugin can
 ### Dev server
 
 
-lit-html is packaged as JavaScript modules. Many developers prefer to import modules using node-style module identifiers, which aren't supported yet by browsers. To run in the browser, these module identifiers need to be transformed to browser-ready module identifiers. The Polymer dev server, which is part of the Polymer CLI, performs this transformation on the fly, so you can preview your project during development without a build step. Another alternative is the Open Web Components Dev Server (`owc-dev-server`).
+lit-html is packaged as JavaScript modules. Many developers prefer to import modules using node-style module identifiers, which aren't supported yet by browsers. To run in the browser, these module identifiers need to be transformed to browser-ready module identifiers. [ES dev server](https://open-wc.org/developing/es-dev-server.html) is an open-source dev server that handles this and other transforms.
 
-If you're using webpack for your build process, you can also use the webpack dev server.
+You can also use the Polymer dev server, which is part of the Polymer CLI, if you already have it installed. For new projects, we recommend the ES dev server.
 
-#### Polymer Dev Server
+If you already have a dev server that integrates with your build process, you can use that, instead.
 
-Install the Polymer CLI:
+#### ES Dev Server
 
-```bash
-npm i -g polymer-cli
-```
+The ES dev server handles rewriting node-style modules to browser-style modules, and enables a build-free development process.
```

Comment:
To me "rewriting node-style modules to browser-style modules" implies rewriting CJS to standard JS modules. es-dev-server just rewrites the import specifiers from bare names to valid paths/URLs. The input is otherwise browser-style already. For IE11 it'll also transform the modules to SystemJS since IE11 doesn't support modules at all.